src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,365 @@
+---
+title: First Discovery Server
+layout: default
+category: Overview
+---
+
+## Launching ##
+
+The First Discovery Server can be launched as a [Kettle](https://github.com/amb26/kettle/blob/KETTLE-32/README.md) application by making use of [Kettle Configs](https://github.com/amb26/kettle/blob/KETTLE-32/README.md#structure-of-a-kettle-config).
+
+There are two typical ways of launching a Kettle app, programmatically and from command line.
```

Comment:
It might be more helpful if the text and the code snippets here, in the FD Server docs, were written a bit more specifically relevant to the FD server itself. The code snippets are currently perfectly generic, but since we know that the reader is working with the FD server (and not just any old Kettle app), perhaps the code snippets could be tweaked to be more specific?